pkg/pretty_errors/prettifier_test.go

```diff
@@ -0,0 +1,57 @@
+package pretty_errors
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+type errTooPretty struct {
+}
+
+func (e errTooPretty) Error() string { return "too pretty" }
+func (e errTooPretty) Prettify() Error {
+	return Error{
+		Original: e,
+		Message:  "too pretty. Expected: less pretty",
+		Details: nil,
+		Code:     "TooPretty",
+	}
+}
+
+func TestPrettifierStrategy_Match(t *testing.T) {
```

Comment:
I think it is best if we could keep the test names consistent. I have not been using lowercases in any of my test names.

Reply:
What do you mean? Do you mean "camel case"? Or underscore? That's strange because that's how the default go tools generate test functions. Here is the common pattern: `Test<StructureName>_<MethodName>` for methods and `Test<FunctionName>` for functions which are not members of a structure. What name do you suggest?

Reply:
I mean to use "camel case" instead of using underscores.

Reply:
but that's the go standard for writing tests on structure methods